res/zee-worker.js

```diff
@@ -1,3 +1,6 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
I'll remove this header. I think zlib doesn't require an acknowledgment that we're using it (but we can still add it like is done in about:license in Firefox I guess).